description.md

```diff
@@ -13,6 +13,46 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Media
 
+The `media` content type refers to audio, visual or audiovisual educational materials.
+Images, videos, recordings, educational video games represent examples of such content.
+
+The media content type represents an easily digestible format for presenting information.
+As such, media is typically embbeded within the text that is destined reading, slides, demos, practice items and projects.
+
+When developing `media` items, it is important to follow the guidelines below:
+
+- use an open source or free development tool.
+- store both the output (the actual image, video etc.) and the editable building blocks.
+
+#### Images
+
+For the creation of images and diagrams we use [draw.io](draw.io).
+`draw.io` is a free, easy to use tool which is available in the browser.
+The resulting images can be exported in different formats, such as PDF, PNG or SVG.
+When storing images, it is important that both the source file (with the extension `drawio` or `svg`) and the immutable output (PDF, PNG, JPG) are stored.
+Note that other tools for the generation of images may be used as long as they are free and provide the source file.
+
+#### Audio-video recordings
+
+Audio or video content is by its nature hard to modify.
+Typically, if a video/audio recording requires alterations, it is re-recorded.
+As such, the video/audio recording is stored as is and if modifications are required, it needs to be recreated.
+In terms of the tools that can be used to create such recordings, given the variety of options, we do not give any specific recommendations.
+Any device that can record at a decent quality is fine.
+
+There is a particular type of video that can be created without using a camera.
+By putting multiple similar images in a slideshow and automating the presentation, the video effect can be achieved.
+This sort of video is useful for educational material that presents processes or phenomenons step by step.
+To create such videos, we need to first create the images and then bundle them up using [ffmpeg](https://ffmpeg.org/).
+In this situation the image source files, the actual images and the video need to be stored in the content repository.
+
+#### Interactive applications
```

Comment:
```suggestion
#### Interactive Applications
```

Reply:
Not sure if this should be part of the media section. This is more of "support software". But, then again, I can't think of a better place to add this content type.